src/core.py

```diff
@@ -105,6 +105,7 @@ def on_config(self, config):
 
         config["markdown_extensions"].append("markdown_inline_graphviz")
         config["markdown_extensions"].append("plantuml_markdown")
+        config["markdown_extensions"].append("mdx_truly_sane_lists")
```

Comment:
I realize we're not actually running tests on build in this repo right now (I can fix in a follow-up), but would you mind adding a check for this in `test_core.py`?

You should be able to run the tests like so: `python -m unittest src/test_core.py`